etcdmain/etcd.go

```diff
@@ -53,7 +62,17 @@ func Main() {
 
 	var stopped <-chan struct{}
 
-	shouldProxy := cfg.isProxy()
+	if cfg.dir == "" {
+		cfg.dir = fmt.Sprintf("%v.etcd", cfg.name)
+		log.Printf("etcd: no data-dir provided, using default data-dir ./%s", cfg.dir)
+	}
+
+	which := identifyDataDirOrDie(cfg.dir)
+	if which != dirEmpty {
+		log.Printf("etcd: %v dir already exists, starting etcd %v...", which, which)
```

Comment:
log.Printf("etcd: dir %s/%v already exists, starting etcd %v...", cfg.dir, which, which)
user can have idea which dir etcd is talking about